src/assets/action.scss

```diff
@@ -90,7 +90,7 @@
 
 		// long text area
 		p {
-			width: 150px;
+			width: 185px;
```

Comment:
Make sure @jancborchardt is ok with this :)

Reply:
:ping_pong: @jancborchardt